### ISSUE TITLE
Error with 2.18.3

### ISSUE THREAD

**Issue by houlia** [WeakAuras/WeakAuras2#2442]
## Describe the bug

**Do you have an error log of what happened?**

```
Date: 2020-09-03 22:08:49
ID: 1
Error occured in: Global
Count: 1
Message: ...s\WeakAuras\Libs\LibGetFrame-1.0\LibGetFrame-1.0.lua line 230:
   bad argument #1 to 'pairs' (table expected, got nil)
Debug:
   [C]: pairs()
   ...s\WeakAuras\Libs\LibGetFrame-1.0\LibGetFrame-1.0.lua:230: GetUnitFrame()
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:370:
      ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:366
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:521: growFunc()
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:1142: DoPositionChildren()
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:1023: PositionChildren()
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:1004: SortUpdatedChildren()
   ...erface\AddOns\WeakAuras\RegionTypes\DynamicGroup.lua:811: Resume()
   WeakAuras\WeakAuras.lua:6412:
      WeakAuras\WeakAuras.lua:6371
   WeakAuras\WeakAuras.lua:6488: UpdatedTriggerState()
   WeakAuras\BuffTrigger2.lua:1357:
      WeakAuras\BuffTrigger2.lua:1349
   WeakAuras\BuffTrigger2.lua:1673:
      WeakAuras\BuffTrigger2.lua:1666
Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"
 = <function> defined =[C]:-1

AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AraBrokerGuildFriends, vr81
  Atlas, vv1.47.03.11304.00
  AtlasClassicWoW, vv1.47.02.11304.00
  AtlasLootClassic, vv1.5.5
  AtlasLootClassicData, vv1.5.5
  AtlasLootClassicDungeonsAndRaids, vv1.5.5
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  Auctionator, v100.0.9
  BagBrother, v
  Bagnon, v8.3.8
  BCT, v1.37
  BeanCounter, v8.2.6434 (SwimmingSeadragon)
  BigWigs, vv15-classic
  BigWigsAhnQirajRuins, v
  BigWigsCore, v
  BigWigsPlugins, v
  BootyBayGazette, v6.2.20200831
  BrokerDurabilityInfo, vv1.29
  BrokerMoneyFu, v80205-2
  BrokerPortals, v1.10.4
  ChocolateBar, vv3.3.4
  ClassicCastbars, vv1.3.11
  ClassicCastbarsOptions, v
  Clique, vv1.0.3-classic
  CommunityDKP, v3.2.2
  CoordsTracking, vv2.15.11305.00
  DBMStatusBarTimers, v
  Details, v
  DetailsTinyThreat, v
  Dominos, v8.3.12
  DominosCast, v
  DominosProgress, v
  DominosRoll, v
  Enchantrix, v8.2.6428 (SwimmingSeadragon)
  EnchantrixBarker, v8.2.6469 (SwimmingSeadragon)
  ExRT, v4160
  ExtendedCharacterStats, v2.5.0
  FishingBuddy, v0.7.3 Beta 7
  GlobalIgnoreList, v1.13.3
  Grid2, vr966
  Grid2LDB, vr966
  Grid2RaidDebuffs, vr966
  GTFO, v4.55
  InFlightLoad, v1.13.005
  Informant, v8.2.6374 (SwimmingSeadragon)
  ItemRack, v
  LeatrixPlus, v1.13.77
  NauticusClassic, v1.3.1
  NovaWorldBuffs, v1.79
  PostalClassic, vv3.6.2
  Prat30, v3.8.26
  Prat30Libraries, v
  Questie, v6.0.5
  RareAware, v1.0
  Regen, v0.1.4
  SaladCthun, v1.01
  ShadowedUnitFrames, vv4.2.3-Classic
  SkilletClassic, v1.16
  StealYourCarbon, v
  Stubby, v8.2.6376 (SwimmingSeadragon)
  ThreatClassic2, v2.23
  TinyTooltip, v8.2.1
  TradeSkillWhisperLookup, v0.2.6-beta
  WaterDispenser, v1.10
  WeakAuras, v2.18.3
  WIM, v3.8.14
  zzInfoclock, v
  BlizRuntimeLib_enUS v1.13.5.11305 <none>
  (ck=850)


```

### To Reproduce
updated to 2.18.3-classic - this is an on-load error

**Screenshots**

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

**Which version of WeakAuras are you using?**

**Are you on World of Warcraft Classic or Retail?**

- [ [x]](url) Classic
- [ ] Retail
- [ ] Shadowlands

**Was it working in a previous version? If yes, which was the last good one?**

### Additional Info

[WeakAurasLua.zip](https://github.com/WeakAuras/WeakAuras2/files/5172366/WeakAurasLua.zip)

**Comment by Krattan** [comment]
I am getting the same crash but with a different trigger
Stacktrace:

1x ...s\WeakAuras\Libs\LibGetFrame-1.0\LibGetFrame-1.0-21.lua:230: bad argument #1 to 'pairs' (table expected, got nil)
[C]: in function `pairs'
...s\WeakAuras\Libs\LibGetFrame-1.0\LibGetFrame-1.0-21.lua:230: in function `GetUnitFrame'
WeakAuras\WeakAuras-2.18.3.lua:7362: in function <WeakAuras\WeakAuras.lua:7325>
WeakAuras\WeakAuras-2.18.3.lua:7425: in function `AnchorFrame'
...aceWeakAuras\RegionTypes\RegionPrototype.lua:857: in function `Expand'
WeakAuras\WeakAuras-2.18.3.lua:6324: in function <WeakAuras\WeakAuras.lua:6318>
WeakAuras\WeakAuras-2.18.3.lua:6404: in function <WeakAuras\WeakAuras.lua:6371>
WeakAuras\WeakAuras-2.18.3.lua:6488: in function `UpdatedTriggerState'
WeakAuras\BuffTrigger2.lua:1357: in function <WeakAuras\BuffTrigger2.lua:1349>
WeakAuras\BuffTrigger2.lua:1673: in function <WeakAuras\BuffTrigger2.lua:1666>

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"
 = <function> defined =[C]:-1

Addons:
$ ls AddOns/
'!BugGrabber'/                      DBM-CustomSounds/      HealBot_Tips/                Questie/
'!MoncaiCompare'/                   DBM-DefaultSkin/       HealBot_tw/                  ShadowedUF_Options/
+Wowhead_Looter/                    DBM-GUI/               ItemRack/                    ShadowedUnitFrames/
AtlasLootClassic/                   DBM-MC/                ItemRackOptions/             TalentSequence/
AtlasLootClassic_Collections/       DBM-Naxx/              Leatrix_Maps/                TipTac/
AtlasLootClassic_Crafting/          DBM-Onyxia/            Leatrix_Plus/                TipTacItemRef/
AtlasLootClassic_Data/              DBM-Party-Classic/     MikScrollingBattleText/      TipTacOptions/
AtlasLootClassic_DungeonsAndRaids/  DBM-StatusBarTimers/   Misc/                        TitanClassic/
AtlasLootClassic_Factions/          DBM-ZG/                MoreStopwatches/             TitanClassicAmmo/
AtlasLootClassic_Maps/              Details/               MSBTOptions/                 TitanClassicBag/
AtlasLootClassic_Options/           Details_DataStorage/   NeatPlates/                  TitanClassicClock/
AtlasLootClassic_PvP/               Details_Streamer/      NeatPlates_Alvara/           TitanClassicGold/
aux-addon/                          Details_TinyThreat/    NeatPlates_BlizzardPlates/   TitanClassicLocation/
BagBrother/                         DropTheCheapestThing/  NeatPlates_ClassicPlates/    TitanClassicLootType/
Bagnon/                             GettingThingsHealed/   NeatPlates_Graphite/         TitanClassicPerformance/
Bagnon_Config/                      GrimoireKeeper/        NeatPlates_Grey/             TitanClassicRegen/
Bagnon_GuildBank/                   HealBot/               NeatPlates_Neon/             TitanClassicRepair/
Bagnon_VoidStorage/                 HealBot_br/            NeatPlates_Quatre/           TitanClassicVolume/
Bartender4/                         HealBot_cn/            NeatPlates_Renaitre/         TitanClassicXP/
BugSack/                            HealBot_Data/          NeatPlates_Roth/             tullaCC/
CEPGP/                              HealBot_de/            NeatPlates_Simple/           ViragDevTool/
CharacterStatsClassic/              HealBot_en/            NeatPlates_Slim_Horizontal/  WeakAuras/
ClassicAuraDurations/               HealBot_es/            NeatPlates_Slim_Vertical/    WeakAurasArchive/
ClassicCodex/                       HealBot_ExtraSkins/    NeatPlatesHub/               WeakAurasCompanion/
ClassTrainerPlus/                   HealBot_fr/            NeatPlatesWidgets/           WeakAurasModelPaths/
ColorPickerPlus/                    HealBot_gr/            PoliteWhisper/               WeakAurasOptions/
DBM-AQ20/                           HealBot_hu/            Prat-3.0/                    WeakAurasTemplates/
DBM-AQ40/                           HealBot_it/            Prat-3.0_Libraries/          WhatsTraining/
DBM-Azeroth/                        HealBot_kr/            Quartz/
DBM-BWL/                            HealBot_Options/       QuestFrameFixer/
DBM-Core/                           HealBot_ru/            QuestIconDesaturation/

Observations:
It seems to happen whenever I apply a buff to myself.

**Comment by mrbuds** [comment]
This is fixed upstream, WeakAuras needs a new tag to include fixed LibGetFrame

**Comment by mrbuds** [comment]
fixed in WeakAuras 2.18.4
